go/stellar/stellarsvc/frontend.go

```diff
@@ -443,35 +434,11 @@ func (s *Server) CancelPaymentLocal(ctx context.Context, arg stellar1.CancelPaym
 	}
 	relay := details.Summary.Relay()
 	dir := stellar1.RelayDirection_YANK
-	return stellar.Claim(ctx, s.G(), s.walletState, relay.KbTxID.String(), relay.FromStellar, &dir, nil)
-}
-
-type balanceList []stellar1.Balance
-
-// Example: "56.0227002 XLM + more"
-func (a balanceList) balanceDescription() (res string, err error) {
```

Comment:
weird. this doesn't appear to be called by anything, and there's an identical method name and implementation in another file. huh.

Reply:
I moved it and forgot to delete this...